controllers/videos.js

```diff
@@ -0,0 +1,60 @@
+var Videos = require("../models/videos");
+
+var VideosController = {
+
+  getVideos: function (request, response, next) {
+    Videos.all(function(error, videos) {
+      if(error) {
+        var err = new Error
+        err.status = 500;
+        err.error = "Error retrieving video rentals list."
+        response.json(err)
+      } else {
+        response.json(videos)
+      }
+    });
+  },
+
+  // video id, sort column, offset (p and n)
+  getVideosSorted: function(request, response) {
+    Videos.sort(request.params.column, request.query.p, request.query.n, function(error, videos) {
+      if(error) {
+        var err = new Error
+        err.status = 404;
+        err.error = "Not found :("
+        response.json(err)
+      } else {
+          response.json(videos)
+      }
+
+    })
+  },
+
+  getVideo: function (request, response) {
+    Videos.find(request.params.title, function(error, video) {
```

Comment:
Are you only expecting a single video for this method? If so, should it be a FindOne method?